product_docs/docs/pgd/5/routing/index.mdx

```diff
@@ -23,6 +23,32 @@ The PGD Proxy connects to one of the EDB Postgres Distributed nodes and monitors
 routing configuration changes as decided by the EDB Postgres Distributed cluster.
 It ensures that the connections are routed to the correct nodes consistently.
 
+The information about currently selected write and read nodes is visible in
+`bdr.node_group_routing_summary`. Note that this is node-local view, the proxy
+always reads from Raft leader to get current and consistent view.
+
+## Leader selection
+
+The write leader is selected by the current Raft leader (either group one or global one,
```

Comment:
local and global? The language previously mandated was sub-group and top-level. The reason for not saying local and global was that the group may not be geo-local and the cluster may not be globally distributed.

```suggestion
The write leader is selected by the current Raft leader (either subgroup one or top-level group one,
```

Reply:
Sure I don't mind, I am used to Kubernetes and TPA calling this global and local

Reply:
I understand completely.... (and I'd love to switch to local and global for simplicity, but I'll keep this conistent with the other content.